br/backup-and-restore-faq.md

```diff
@@ -8,6 +8,23 @@ aliases: ['/docs/dev/br/backup-and-restore-faq/']
 
 This document lists the frequently asked questions (FAQs) and the solutions about Backup & Restore (BR).
 
+If you encountered problems that are not listed in this document and cannot be resolved, you are welcome to ask your questions to [AskTUG](https://asktug.com/) community.
+
+## In TiDB v5.4.0 and later versions, when backup tasks perform on the cluster under high workload, why does the speed of backup tasks become slow?
+
+Starting from TiDB v5.4.0, TiKV introduces the auto-tune feature for backup tasks. For clusters in v5.4.0 or later versions, this feature is enabled by default. When the cluster workload is high, the feature limits the resources used by backup tasks to reduce the impact on the online cluster. For more information, refer to [BR Auto-Tune](/br/br-auto-tune.md).
```

Comment:
IMHO, "heavy workload" sounds better.
<img width="253" alt="image" src="https://user-images.githubusercontent.com/36239017/150281599-3cff2e92-df57-4848-873f-9b34a98c7145.png">

(Seems there are other "high workloads" in the document, ditto if possible.)

```suggestion
Starting from TiDB v5.4.0, TiKV introduces the auto-tune feature for backup tasks. For clusters in v5.4.0 or later versions, this feature is enabled by default. When the cluster workload is heavy, the feature limits the resources used by backup tasks to reduce the impact on the online cluster. For more information, refer to [BR Auto-Tune](/br/br-auto-tune.md).
```